### DISCUSSION TITLE
Feature request: allow reverse datetime sorting in Notifications view

### DISCUSSION THREAD

**Discussion by peterazmanov** [community/community#5814] [Other Feature Feedback, Questions, & Ideas]
I haven't found a way to view notifications from the selected repo in the "Oldest first" order. In order to do so I need to click `Next` multiple times to reach last page and start viewing issues from the bottom up... After checking as `Done` all notifications on the last page I need to proceed to the previous but it's not working properly (see https://github.com/github/feedback/discussions/3605#discussioncomment-1386535).

It would be great to have a "Sort by..." button or smth similar.

**Comment by hughlilly** [comment]
(Probably) Related to this, when viewing one’s own repositories there are three sort options:

<img width="319" alt="image" src="https://user-images.githubusercontent.com/12046008/140811599-34525533-9c4f-4c10-adb0-ec6957efc889.png">

Could we get a double arrow button—`↑↓` or perhaps 🔃; I don’t know what the convention is—to reverse the sort? It could live next to this text:
>**9** results for **forked** repositories sorted by **last updated** 🔃

**Comment by edgarrmondragon** [comment]
Bump. This would be nice to tackle my notifications in a FIFO fashion 😄

**Comment by yguedidi** [comment]
+1
Would be great to process lots of notification after for example a period of holidays :)
